spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,10 @@
+RSpec.describe 'Food Services' do
+  describe 'food_search(food)' do
+    it 'returns relevant foods' do
+      search = FoodService.call_for_a_food("sweet potatoes")
+      expect(search).to be_a(Hash)
+      expect(search[:results]).to be_an(Array)
```

Comment:
This is on the right track, but we would also want to dig into the `search[:results]` array a bit deeper for all of the info that we plan on storing in our Food objects as well as the data type of `total_results`.